src/transformers/quantizers/quantizer_awq.py

```diff
@@ -95,6 +98,8 @@ def _process_model_after_weight_loading(self, model):
             model = fuse_awq_modules(model, self.quantization_config)
             model._awq_is_fused = True  # TODO: consider storing this flag in model.config instead
 
+        model._is_quantized_training_enabled = self.is_trainable
```

Comment:
In #28991 this property becomes deprecated. We should use `model.hf_quanitzer.is_trainable` instead

Reply:
ah yes good catch ! Will rebase once I merge #28991 then!

Reply:
Done ! Thanks for the catch !